products/db-checkbook/build_scripts/build.py

```diff
@@ -195,21 +193,26 @@ def assign_category(row):
     gdf['final_category'] = gdf[cols].apply(lambda row: assign_category(row), axis=1)
     return gdf
 
-def run_build() -> pd.DataFrame:
+def run_build() -> None:
     """
     :return: historical spending data
     """
+    print("merge and group source data ...")
```

Comment:
do we want to keep these print statements? (I am pro-print statement, just wondering if leaving these here was intentional or not)

Reply:
also pro-print statements, I think they are helpful for knowing where in the process we are

Reply:
these I'd meant to keep, they can definitely get to be a lot but with build pipelines I especially like logging when a major stage of it has started

I resist putting these types of "status" print statements in smaller functions in favor of putting them in the higher level calling functions